Resources/Prototypes/CharacterItemGroups/scienceGroups.yml

```diff
@@ -0,0 +1,103 @@
+- type: characterItemGroup
+  id: LoadoutUniformsScience
+  items:
+    - type: loadout
+      id: LoadoutScienceUniformJumpskirtSenior
+    - type: loadout
+      id: LoadoutScienceUniformJumpsuitSenior
+    - type: loadout
+      id: LoadoutScienceUniformJumpskirtRoboticist
+    - type: loadout
+      id: LoadoutScienceUniformJumpsuitRoboticist
+    - type: loadout
+      id: LoadoutScienceUniformJumpsuitMonasticRobeDark
+    - type: loadout
+      id: LoadoutScienceUniformJumpsuitMonasticRobeLight
+
+- type: characterItemGroup
+  id: LoadoutOuterScience
+  items:
+    - type: loadout
+      id: LoadoutScienceOuterCoat
+    - type: loadout
+      id: LoadoutScienceOuterLabcoat
+    - type: loadout
+      id: LoadoutSciencegOuterCoatRobo
+    - type: loadout
+      id: LoadoutScienceOuterWinterSci
+    - type: loadout
+      id: LoadoutScienceOuterLabcoatSeniorResearcher
+    - type: loadout
+      id: LoadoutScienceOuterExplorerLabcoat
+    - type: loadout
+      id: LoadoutOuterRobeTechPriest
+    - type: loadout
+      id: LoadoutOuterPlagueSuit
+    - type: loadout
+      id: LoadoutOuterNunRobe
+    - type: loadout
+      id: LoadoutOuterHoodieBlack
+    - type: loadout
+      id: LoadoutOuterHoodieChaplain
+    - type: loadout
+      id: LoadoutScienceOuterWinterCoatMantis
+
+- type: characterItemGroup
+  id: LoadoutGlovesScience
+  items:
+    - type: loadout
+      id: LoadoutScienceHandsGlovesColorPurple
+    - type: loadout
+      id: LoadoutScienceHandsGlovesLatex
+    - type: loadout
+      id: LoadoutScienceHandsGlovesRobohands
+
+- type: characterItemGroup
+  id: LoadoutNeckScience
+  items:
+    - type: loadout
+      id: LoadoutScienceNeckTieSci
+    - type: loadout
+      id: LoadoutScienceNeckScarfStripedPurple
+    - type: loadout
+      id: LoadoutScienceNeckStoleChaplain
+    - type: loadout
+      id: LoadoutScienceNeckScarfStripedBlack
+
+- type: characterItemGroup
+  id: LoadoutMaskScience
+  items:
+    - type: loadout
+      id: LoadoutScienceMaskPlague
+
+- type: characterItemGroup
+  id: LoadoutHeadScience
+  items:
+    - type: loadout
+      id: LoadoutScienceHeadHatBeret
+    - type: loadout
+      id: LoadoutHeadHoodTechPriest
+    - type: loadout
+      id: LoadoutScienceHeadHatFez
+    - type: loadout
+      id: LoadoutScienceHeadHatHoodNunHood
+    - type: loadout
+      id: LoadoutScienceHeadHatPlaguedoctor
+    - type: loadout
+      id: LoadoutScienceHeadHatWitch
+    - type: loadout
+      id: LoadoutScienceHeadHatWitch1
+
+- type: characterItemGroup
+  id: LoadoutEyesScience
+  items:
+    - type: loadout
+      id: LoadoutScienceEyesHudDiagnostic
+    - type: loadout
+      id: LoadoutScienceEyesEyepatchHudDiag
+
+- type: characterItemGroup
+  id: LoadoutShoesScience
+  items:
+    - type: loadout
+      id: LoadoutScienceShoesBootsWinterSci
```

Comment:
```suggestion
      id: LoadoutScienceShoesBootsWinterSci

```

Reply:
Did you really have to make a review to add a newline at the end of the file